apps/shared/avifutil.h

```diff
@@ -10,6 +10,59 @@
 extern "C" {
 #endif
 
+// NEXTARG(), MAIN(), INIT_ARGV(), FREE_ARGV() for UTF8-aware command line parsing.
+#if defined(_WIN32)
+#define NEXTARG()                                                      \
+    if (((argIndex + 1) == argc) || (argv[argIndex + 1][0] == L'-')) { \
+        fprintf(stderr, "%s requires an argument.", arg);              \
+        goto cleanup;                                                  \
```

Comment:
IMPORTANT: set `returnCode` to 1 before `goto cleanup`. Make the same change to line 25.

Reply:
Right, I fixed this logic in https://github.com/AOMediaCodec/libavif/pull/1699